openpype/tools/publisher/window.py

```diff
@@ -64,19 +64,16 @@ def __init__(self, parent=None, controller=None, reset_on_show=None):
         if reset_on_show is None:
             reset_on_show = True
 
-        if parent is None:
-            on_top_flag = QtCore.Qt.WindowStaysOnTopHint
-        else:
-            on_top_flag = QtCore.Qt.Dialog
-
-        self.setWindowFlags(
-            QtCore.Qt.WindowTitleHint
+        flags = (
```

Comment:
Before we weren't storing them in a variable, and we don't seem to do that in other files.